core_ui/src/main/java/com/trempel/core_ui/BindingAdapters.kt

```diff
@@ -58,4 +58,15 @@ internal fun setRecyclerViewItems(recyclerView: RecyclerView, items: List<Recycl
         recyclerView.adapter = adapter
     }
     adapter.submitList(items.orEmpty())
+}
+
+
+@BindingAdapter("app:activeBtn")
+fun View.activeBtn(quantity: Int) {
```

Comment:
please rename the function and related binding attribute to something connected with minimum value